Makefile

```diff
@@ -1,3 +1,4 @@
+.PHONY: all pdf mostlyclean clean
```

Comment:
There are `.PHONY` targets at the bottom of the makefile.  Putting them before the `all` target breaks makes that don't understand `.PHONY`.  The existing `.PHONY` line at the bottom suffices, no?